src/v2/Apps/Conversation/Components/Conversation.tsx

```diff
@@ -134,6 +161,33 @@ const Conversation: React.FC<ConversationProps> = props => {
     orderID: orderID,
   })
 
+  // New Messages
+  const isBottomVisible = useOnScreen(bottomOfPage)
```

Comment:
Checking if the bottom is visible - or the user has scrolled to know when to display the cta.